packages/react-navigation/src/__tests__/rum/instrumentation/DdRumReactNavigationTracking.test.tsx

```diff
@@ -7,17 +8,25 @@
 import { InternalLog } from '@datadog/mobile-react-native/internal';
 import { DdRum } from '@datadog/mobile-react-native';
 import type { NavigationContainerRef, Route } from '@react-navigation/native';
-import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
 import { render, fireEvent } from '@testing-library/react-native';
 import mockBackHandler from 'react-native/Libraries/Utilities/__mocks__/BackHandler.js';
-import { View, Text, Button, AppState, BackHandler } from 'react-native';
-import React from 'react';
+import { AppState, BackHandler } from 'react-native';
+import React, { createRef } from 'react';
 
 import type { ViewNamePredicate } from '../../../rum/instrumentation/DdRumReactNavigationTracking';
 import { DdRumReactNavigationTracking } from '../../../rum/instrumentation/DdRumReactNavigationTracking';
 
 import { AppStateMock } from './__utils__/AppStateMock';
+import {
+    FakeNavigator1,
+    FakeNavigator2,
+    FakeNestedNavigator
+} from './__utils__/Navigators/NavigatorsV5';
+
+// TODO: inject this as a global
```

Comment:
This function will be available as `jest.mocked` in jest v27. However jest is still v26 in the latest React Native, so the switch to v26 to v27 might not be easy.

Exporting this as a global is not easy also, so I'll try to upgrade jest the next time I need this variable